bootstrap/src/contexts/Auth/state.js

```diff
@@ -0,0 +1,76 @@
+// (C) 2019 GoodData Corporation
+import { useState } from "react";
+
+export const AuthStatus = {
+    AUTHORIZING: "AUTHORIZING",
+    AUTHORIZED: "AUTHORIZED",
+    UNAUTHORIZED: "UNAUTHORIZED",
+    LOGGING_IN: "LOGGING_IN",
+    LOGGING_OUT: "LOGGING_OUT",
+    REGISTERING: "REGISTERING",
```

Comment:
Registering is not supported in this app. Please remove the mentions of registration.